cli/server.ts

```diff
@@ -73,9 +73,13 @@ const expectedConfig = ["_", "config", "mock-driver"];
   });
 
   driver.on("driver ready", () => {
```

Comment:
Let's make this callback `async` so we can use `await` instead of promises.

Reply:
@balloob AFAIF it's not good practise to use async functions in event listeners as the EE doesn't handle the rejections. Even if we would be 100% sure it doesn't reject

Reply:
You can use try…catch. It behaves just like `.catch()` on promises.

```ts
try {
  await server.start()
  console.info("Server listening on port 3000");
} catch (err) {
  console.error("Unable to start server", err);
  await driver.destroy();
}
```

Reply:
I know, in other repo i maintain some user told me that is better to avoid

```js
something.on('eventName', async function(){})
```

It's just a bad pratcise even if you are sure (by using a try catch for example) that function never throws. Anyway will change it no problem :)